Terminal.Gui/View/Layout/ViewLayout.cs

```diff
@@ -858,12 +778,6 @@ internal void CollectAll (View from, ref HashSet<View> nNodes, ref HashSet<(View
         foreach (View? v in from.InternalSubviews)
```

Comment:
I know it wasn't part of this change set, but it's something simple that doesn't need a whole issue to track:

The body of this loop needs to guard against nulls.

Also, we should use a different variable name, instead of `from`, as that's a keyword for C# LINQ statements.

Also a big thread safety problem here, but that's a more ubiquitous thing that _will_ need dedicated issues, thought, and work to handle appropriately.

Reply:
Theoretically yes. But this code has existed in TG since the beginning of time and there's never been a null issue. 

Threading yes.

Suggest a better name for `from` and I'll change it.

Reply:
> Suggest a better name for `from` and I'll change it.

Hmm.  Maybe `source` or something of that nature?

Reply:
In the end, it's a method parameter, so it's about a 0.1 on a scale of 0 to 10 for importance. 🤷‍♂️

If it stays `from`, though, it should probably be prefixed with `@` for formality.